src/command_modules/azure-cli-storage/tests/test_storage_blob_scenarios.py

```diff
@@ -70,55 +65,51 @@ def test_storage_blob_upload_64mb_file(self, resource_group, storage_account):
 
     def verify_blob_upload_and_download(self, group, account, file_size_kb, blob_type,
                                         block_count=0, skip_download=False):
-        container = self.create_random_name(prefix='cont', length=24)
         local_dir = self.create_temp_dir()
         local_file = self.create_temp_file(file_size_kb)
         blob_name = self.create_random_name(prefix='blob', length=24)
-        account_key = self.get_account_key(group, account)
+        account_info = self.get_account_info(group, account)
 
-        self.set_env('AZURE_STORAGE_ACCOUNT', account)
-        self.set_env('AZURE_STORAGE_KEY', account_key)
+        container = self.create_container(account_info)
 
-        self.cmd('storage container create -n {}'.format(container))
+        self.storage_cmd('storage blob exists -n {} -c {}', account_info, blob_name, container)
```

Comment:
Is there a reason you removed the check from here?  If not to check that the output is false, you could just remove this.